modules/acs-architecture-connections.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * architecture/acs-architecture.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-architecture-connections_{context}"]
+= {product-title-short} connection protocols and default ports
+
+The protocols and default ports for connections within {product-title-short} and between {product-title-short} and external components are shown in the following table.
+
+.{product-title-short} connections between components
+
+[cols="3",options="header"]
+|===
+| Component or external entity | Connection type | Port
+
+| Central and external services, for example, Jira and PagerDuty
+| HTTPS
+| ?
+
+| Central and secured cluster
+| HTTPS
+| 443
+
+| Central and sensor
+| TCP
+| 443
+
+| Central and user ({product-title-short} web portal or REST API)
```

Comment:
Users may use HTTPS, but I believe gRPC is allowed, too, if the user wanted to (at the same port)